webpack.config.js

```diff
@@ -8,11 +8,22 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const AngularCompilerPlugin = require('@ngtools/webpack').AngularCompilerPlugin;
 const pjson = require('./package.json');
+const config = require('./config.js')
 
 if (process.env.NODE_ENV == null) {
     process.env.NODE_ENV = 'development';
 }
-const ENV = process.env.ENV = process.env.NODE_ENV;
+
+const NODE_ENV = process.env.NODE_ENV;
+const envConfig = config.load(process.env.ENV)
+config.log(envConfig)
+
+if (process.env.ENV == null) {
+    // Defaults to development if an ENV is not explicitly chosen. Also preserves the selfHost option
+    process.env.ENV = 'development' 
+} 
+
+const ENV = process.env.ENV;
```

Comment:
It this used anywhere?

Reply:
It is used once on line 137. I'm not sure exactly what that `DefinePlugin` is used for but it looked like that was what the rest of the code was going to use so I wanted to expose the ENV environments vs just the NODE_ENV environments.

Reply:
I believe it should actually be the NODE_ENV environment, since it's now used to determine log level and enable angular production mode.

Reply:
Which should let you remove line 21-26.

Reply:
Perfect! I'll get that switched.